### ISSUE TITLE
[Bug]: Entity id extension

### ISSUE THREAD

**Issue by vasmark1** [GTMEGA/EndlessIDs#157]
### OS

Windows

### GPU

NVIDIA

### Modpack (Optional)

My own

### Game log

[latest.log](https://github.com/GTMEGA/EndlessIDs/files/13693753/latest.log)

### Description

Issue is present only in multiplayer, console is spammed with error messages like "Server attempted to spawn entity with id ...". Only host can see mobs, other players can't.

**Comment by KempeR0744** [comment]
disable entity id extended, in the endlessids config.

**Comment by vasmark1** [comment]
> disable entity id extended, in the endlessids config.

Yup, this fixes problem, just mentioned it here in issue so falsepattern knows about it
